parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -111,7 +111,7 @@ pub type ForeignAssetsConvertedConcreteId = assets_common::ForeignAssetsConverte
 	(
 		// Ignore `TrustBackedAssets` explicitly
 		StartsWith<TrustBackedAssetsPalletLocation>,
-		// Ignore assets that start explicitly with our `GlobalConsensus(NetworkId)`, means:
+		// Ignore asset which starts explicitly with our `GlobalConsensus(NetworkId)`, means:
```

Comment:
But "that" is correct here. (https://www.grammarly.com/blog/which-vs-that/)

Reply:
Nice, though I went to a grammar school they did not teach grammar - for some reason it wasn't cool in the '80s.